web/pdf_viewer.js

```diff
@@ -144,6 +146,7 @@ var PDFViewer = (function pdfViewer() {
     this.renderInteractiveForms = options.renderInteractiveForms || false;
     this.enablePrintAutoRotate = options.enablePrintAutoRotate || false;
     this.renderer = options.renderer || RendererType.CANVAS;
+    this.apiIgnoreErrors = options.apiIgnoreErrors || false;
```

Comment:
Sure, there's no good reason why we cannot default to `true` everywhere in the viewer code; don't know why I didn't do that to be honest :-)